packages/devextreme/testing/styles/jest.config.json

```diff
@@ -1,4 +1,5 @@
 {
+    "testEnvironment": "node",
```

Comment:
Could you explain why this line has to be added here and removed in apps/demos?

https://github.com/DevExpress/DevExtreme/pull/27387/files#r1606752834